src/pages/index.js

```diff
@@ -128,3 +228,43 @@ const enableValidation = (config) => {
 enableValidation(validationSettings);
 
 //initialCards.forEach((cardData) => renderCard(cardData, cardListEl));
+//const cardElement = getCardElement(cardData);
+//cardListEl.prepend(cardElement);
+
+//runs on page load
+// gets user info from server and sets it on the dom
+// api
+//   .getUserInfo()
+//   .then((userData) => {
+//     console.log(userData); //{name: sldjf, about: skdf}
+
+//   })
+// .catch((err) => {
+//   console.error(err);
+//   alert(`Error ${err}. Could not get user info.`);
+// });
+
+const section = new Section(
+  {
+    items: [],
+    renderer: renderCard,
+  },
+  ".cards__list"
+);
+
+//gets the cards off the server
+// api.getInitialCards().then((cards) => {
+
+//   console.log(cards);
+// });
+Promise.all([api.getInitialCards(), api.getUserInfo()])
+  .then(([initialCardData, userData]) => {
+    section.setItems(initialCardData.reverse());
+    section.renderItems();
+    userInfo.setUserInfo({ name: userData.name, description: userData.about });
+    userInfo.changeAvatar(userData.avatar);
+  })
+  .catch((err) => {
+    console.error(err);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors